torcharrow/functional.py

```diff
@@ -109,6 +109,34 @@ def __getattr__(op_name: str):
     setattr(sys.modules["torcharrow.functional"], op_name, wrapper)
     return wrapper
 
+### operations in for text domain
+def add_tokens(
+    input_col: Union[ListColumn, List[Union[int, str]]],
+    tokens: Union[ListColumn, List[Union[int, str]]],
+    begin: bool,
+) -> NumericalColumn:
+    """
+    Append or prepend a list of tokens/indices to a column. 
```

Comment:
Thanks for the nice stub and doc string!

Mind if also add a entry into the functional doc? (e.g. `Text Operations`) : 

https://github.com/pytorch/torcharrow/blame/d20e01b0cbb6cad4be10526344422b0ef6ff4efd/docs/source/functional.rst#L35

Reply:
Yup done!